BlueprintUI/Tests/AttributedTextTests.swift

```diff
@@ -119,9 +119,6 @@ class AttributedTextTests: XCTestCase {
         var text = AttributedText("some emoji: 😵‍💫👨‍👩‍👧‍👦🏃🏽 and some hiragana:  あいうえお and some katakana: アイウエオカキクケコ")
         text.color = .blue
 
-        let partialEmoji = text.range(of: "😵")!
```

Comment:
This was failing on iOS 17 tests and tbh I don't know if it really makes sense so I've removed it.

Reply:
I want to say that we copied these from some tests in the swift repo but I can't find the originals.

Reply:
I know there's one other kind of unusual emoji range test I've run into here: https://github.com/squareup/market/pull/7112/files#r1341394435